src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/waypoints_pending_resumption_handler.cc

```diff
@@ -139,71 +144,61 @@ void WayPointsPendingResumptionHandler::on_event(
     const application_manager::event_engine::Event& event) {
   using namespace application_manager;
   SDL_LOG_AUTO_TRACE();
+  sync_primitives::AutoLock lock(pending_resumption_lock_);
+
+  unsubscribe_from_event(Navigation_SubscribeWayPoints);
```

Comment:
I don't remember if we can send multiple Navigation_SubscribeWayPoints to HMI and wait for more then 1 response. But if yes, unsubscribe from this event will cause missing next responses from HMI

Reply:
As I remember, we cannot, because we send all requests sequentially, one after response to another. And every next request has own `corr_id`, which was initially assigned in HandleResumptionSubscriptionRequest method.